README.md

```diff
@@ -8,15 +8,23 @@ A lightweight notification daemon for Wayland. Works on Sway.
 
 mako implements the [GNOME Desktop Notifications Specification][gnome-draft].
 
-Feel free to join the IRC channel: ##emersion on irc.freenode.net.
+Feel free to join the IRC channel: #emersion on irc.freenode.net.
```

Comment:
The channel name is ##emersion, not #emersion.

Reply:
My bad, though it was a typo :sweat: